istioctl/pkg/auth/upgrader.go

```diff
@@ -162,6 +207,36 @@ func (ug *Upgrader) convert(authzPolicies *model.AuthorizationPolicies) error {
 	return nil
 }
 
+// convertClusterRbacConfig converts ClusterRbacConfig to AuthorizationPolicy.
+func (ug *Upgrader) convertClusterRbacConfig(authzPolicies *model.AuthorizationPolicies) error {
+	clusterRbacConfig := authzPolicies.GetClusterRbacConfig()
+	if clusterRbacConfig == nil {
+		return fmt.Errorf("no ClusterRbacConfig found")
+	}
+	rootNamespace, err := ug.getIstioRootNamespace()
+	if err != nil {
+		return fmt.Errorf("failed to get Istio root namespace: %s", err)
+	}
+	type RbacTmplData struct {
+		RootNamespace string
+	}
+	rootNs := RbacTmplData{
+		RootNamespace: rootNamespace,
+	}
+	var policy string
+	switch clusterRbacConfig.Mode {
+	case rbac_v1alpha1.RbacConfig_OFF:
+		policy, err = fillTemplate(rbacGlobalAllowAll, rootNs)
+	case rbac_v1alpha1.RbacConfig_ON:
+		policy, err = fillTemplate(rbacGlobalDenyAll, rootNs)
```

Comment:
Could you add a TODO for the support of other modes? Maybe better to return error on unsupported modes.

Reply:
I actually have some code locally based on this branch to support namespace-level, so I'll send out another PR after this one has been merged.

Reply:
I'll also add logs for things that we don't support (yet)